src/App/App.js

```diff
@@ -67,6 +77,7 @@ class App extends Component {
 						movie={this.state.movie}
 						statusCode={this.state.statusCode}  
 						error={this.state.error} 
+						movieTrailer={this.state.movieTrailer[0]}
```

Comment:
We can add carousel and change the props to pass down an array in a later iteration :)